components/modal/index.en-US.md

```diff
@@ -65,7 +66,9 @@ The properties of the object are follows:
 | width      | Width of dialog           | string\|number | 416           |
 | iconType   | Type of Icon component    | string | question-circle |
 | okText     | Text of OK button    | string           | OK       |
+| okType    | type of the confirmation button                              | string        | primary   |
 | cancelText | Text of cancel button    | string           | Cancel       |
+| cancelType| type of the confirmation button                              | string        | none   |
```

Comment:
And docs ....

Just `grep -r 'cancelType' ./components` to find all the cancelType and remove it.